File: Top 100 Codes/05-Operations on Strings/11-Remove spaces from a string/Removespacesfromastring.py
def remove_spaces(s):
    result = ""
    for char in s:
        if char != " ":
            result += char
    return result

# Taking input from the user
input_str = input("Enter a string: ")
output_str = remove_spaces(input_str)

print("String without spaces:", output_str)

File: Top 100 Codes/05-Operations on Strings/11-Remove spaces from a string/Removespacesfromastring.cpp
#include <iostream>
#include <string>

using namespace std;

string removeSpaces(string str) {
    string result = "";
    for (int i = 0; i < str.length(); i++) {
        if (str[i] != ' ') {
            result += str[i];
        }
    }
    return result;
}

int main() {
    string input;
    cout << "Enter a string: ";
    getline(cin, input);

    string output = removeSpaces(input);
    cout << "String without spaces: " << output << endl;

    return 0;
}
